viewer/src/views/threejs/cad/CadNode.ts

```diff
@@ -176,33 +176,42 @@ export class CadNode extends THREE.Object3D {
   }
 
   private createLoadSectorsPipeline(): Subject<ThreeCameraConfig> {
+    const suspendLoading = this.loadingHints.suspendLoading;
+
     const loadSectorOperator = flatMap((s: WantedSector) => this._repository.loadSector(s));
     const consumeSectorOperator = flatMap((sector: ParsedSector) => this.rootSector.consumeSector(sector.id, sector));
+    function suspendIfLoadingToggled<T>(): OperatorFunction<T, T> {
+      return mergeMap<T, ObservableInput<T>>(x => (!suspendLoading ? of(x) : empty()));
```

Comment:
This will create new observables that will be subscribed to for each thing that goes through the pipeline. Not too bad if it happens at the top, where we will get one or two new subscriptions every 100 ms, but this causes some serious performance issues when done further down the line. Calls to `observable.subscribe` was what took between 16 and 60 ms now and then in the first implementation, where we had something similar.

Maybe we can achieve something with a filter instead? I think those are evaluated runtime. @j-bjorne: correct me if I am wrong.

Reply:
@j-bjorne : bump

Reply:
Removed suspend.